onnxscript/rewriter/ort_fusions/attention_test.py

```diff
@@ -0,0 +1,156 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+from __future__ import annotations
+
+import unittest
+
+import numpy as np
+import packaging.version
+import parameterized
+
+import onnxscript
+import onnxscript.ir as ir
+import onnxscript.rewriter.ort_fusions._core as xformers
+from onnxscript import FLOAT, script
+from onnxscript import opset18 as op
+from onnxscript.ir.passes.common import shape_inference
+from onnxscript.rewriter.ort_fusions._test_utils import ORT_VERSION, assert_allclose, ort_run
+
+msft_op = onnxscript.values.Opset("com.microsoft", 1)
+
+
+class TestAttentionFusion(unittest.TestCase):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.batchsize = 2
+        self.seqlen = 8
+        self.max_seqlen = 32
+        self.headsize = 16
+        self.num_heads = 10
+        self.input_hidden_size = self.headsize * self.num_heads
+        self.q_hidden_size = 160
+        self.k_hidden_size = 160
+        self.v_hidden_size = 160
+
+    def random_inputs(self, with_past=False):
+        """Generate random inputs for the model."""
+        B = self.batchsize
```

Comment:
Avoid upper case variable names to follow python naming conventions.

Reply:
So, I think it is an acceptable exception to the general rule.

Reply:
Leaving variable names for shapes as they are, as I agree (B, S, ...) for example are commonly used for shape notations and might be easier to recognize in this context.